demosplan/DemosPlanCoreBundle/Twig/Extension/ProcedureExtension.php

```diff
@@ -303,6 +303,8 @@ public function getNameFunction($procedure, $type = 'auto')
         try {
             $procedure = $this->getProcedureObject($procedure);
         } catch (Exception $exception) {
+            $this->logger->error('Could not get procedure object', [$exception]);
```

Comment:
Thank you for the logging, but where does the logger come from?

Reply:
Oh thx my bad, fixed hier https://github.com/demos-europe/demosplan-core/pull/1317/commits/4e372f8364f92a280929e03a41f7726c913cf0cc